frontend/src/app/components/bitcoin-invoice/bitcoin-invoice.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="wrapper">
```

Comment:
This is mostly a copy/paste from the existing `bitcoin-method` component. But unfortunately it depends on too many things to simple move it in this repo. So copy paste is the way for now.